src/client/sandbox/code-instrumentation/index.js

```diff
@@ -31,6 +31,22 @@ export default class CodeInstrumentation extends SandboxBase {
         this.locationAccessorsInstrumentation.attach(window);
         this.elementPropertyAccessors = this.propertyAccessorsInstrumentation.attach(window);
 
+        window[INSTRUCTION.getEval] = ev => {
+            if (ev !== window.eval)
+                return ev;
+
+            return script => {
+                if (typeof script === 'string')
+                    script = processScript(script);
+
+                // NOTE: Assign the eval method to a variable to simulate the original behavior -
+                // eval switches to the global context. (GH-260)
+                var evl = eval;
```

Comment:
We can avoid doing it if we'll use `evalFn`. As I understand `eval` will switch to global context mode if you pass it as parameter (requires check)